docs/content/guides/developer/app-examples/trustless-swap.mdx

```diff
@@ -996,49 +1068,151 @@ The UI design consists of three parts:
-});
+{@inject: examples/trading/frontend/tsconfig.json}
+
+Replace the content of `vite.config.ts` with the following:
 
-ReactDOM.createRoot(document.getElementById('root')!).render(
-	<React.StrictMode>
-		<Theme appearance="light">
-			<QueryClientProvider client={queryClient}>
-				<SuiClientProvider networks={networkConfig} defaultNetwork="testnet">
-					<WalletProvider autoConnect>
-						<RouterProvider router={router} />
-					</WalletProvider>
-				</SuiClientProvider>
-			</QueryClientProvider>
-		</Theme>
-	</React.StrictMode>,
-);
+{@inject: examples/trading/frontend/vite.config.ts}
+
+### Adding Tailwind CSS
+
+This app uses Tailwind CSS for styling. Run the following command to add Tailwind CSS and its dependencies:
+
+```bash
+pnpm add tailwindcss@latest postcss@latest autoprefixer@latest
+```
+
+or
+
+```bash
+yarn add tailwindcss@latest postcss@latest autoprefixer@latest
+```
+
+Next, generate the Tailwind CSS configuration file by running:
+
+```bash
+npx tailwindcss init -p
 ```
 
+Replace the content of `tailwind.config.js` with the following:
+
+{@inject: examples/trading/frontend/tailwind.config.js}
+
+Add the `src/styles/` directory and add `base.css`: 
+
+{@inject: examples/trading/frontend/src/styles/base.css}
+
+### Connecting your deployed package
+
+First, deploy your package via the [scripts in the api directory](#deployment).
+
+Then, create a `src/constants.ts` file and fill it with the following:
+
+{@inject: examples/trading/frontend/src/constants.ts}
+
+### Add helper functions and UI components
+
+Create a `src/utils/` directory and add `helpers.ts`: 
+
+{@inject: examples/trading/frontend/src/utils/helpers.ts}
+
+Create a `src/components/` directory and add the following components: 
+
+- `ExplorerLink.tsx`:
+
+{@inject: examples/trading/frontend/src/components/ExplorerLink.tsx}
+
+- `InfiniteScrollArea.tsx`:
+
+{@inject: examples/trading/frontend/src/components/InfiniteScrollArea.tsx}
+
+- `Loading.tsx`:
+
+{@inject: examples/trading/frontend/src/components/Loading.tsx}
+
+- `SuiObjectDisplay.tsx`:
+
+{@inject: examples/trading/frontend/src/components/SuiObjectDisplay.tsx}
+
+### Set up routes
+
+The imported template only has a single page. To add more pages, you need to set up routing.
+
+First install the necessary dependencies:
+
+```bash
+pnpm add react-router-dom react-hot-toast
+```
+
+or
+
+```bash
+yarn add react-router-dom react-hot-toast
+```
+
+Then, create a `src/routes/` directory and add `index.tsx`. This file contains the routing configuration:
+
+{@inject: examples/trading/frontend/src/routes/index.tsx}
+
+Add the following respective files to the `src/routes/` directory:
+
+- `root.tsx`. This file contains the root component that is rendered on every page:
+
+{@inject: examples/trading/frontend/src/routes/root.tsx}
+
+- `LockedDashboard.tsx`. This file contains the component for the `Manage Objects` page.
+
+```tsx
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
I'm currently updating the plugin to handle this but in the meantime I don't think we need to add the copyright bits. Ashok might have an opinion.